### PR TITLE
Exercise/resistor color trio

### PR DESCRIPTION
**Pull request by rodmagaldi**
I implemented the [resistor-color-trio](https://github.com/exercism/problem-specifications/tree/main/exercises/resistor-color-trio) exercise based on the [contributing guidelines](https://github.com/exercism/typescript/blob/main/CONTRIBUTING.md).

However, I did not understand point 13 of the guideline:

> 13. Create an entry in config.json: a unique new UUID (you can use the configlet uuid tool to generate one, scroll down to tools to see how you can get it), give it a difficulty (should be similar to similar exercises), and make sure the order of the file is sane. Currently the file is ordered first on core - non core, then on difficulty low to high, and finally lexographically.

Is there anything that needs to be done as to that? I looked around in other exercises and there didn't seem to be anything else.

Also, let me know if you catch any mistakes I might have made :laughing: 

Thanks!

### CODE REVIEW

**Comment by rodmagaldi** [comment]
Hey @SleeplessByte ! I added the new exercise to the json file.

Anything else to be done here ?

**Comment by SleeplessByte** [comment]
So sorry @rodmagaldi ! I was swamped with work and then cleared my Notifications and completely missed this.

File: exercises/practice/resistor-color-trio/resistor-color-trio.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function decodedResistorValue() {
+  throw new Error('Delete this and implement the function.')
```
Comment (SleeplessByte) [inline exercises/practice/resistor-color-trio/resistor-color-trio.ts:2]:
```suggestion
  throw new Error('Remove this statement and implement this function')
```

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Do you have Twitter? I wanna shout you out

**Comment by rodmagaldi** [comment]
> Do you have Twitter? I wanna shout you out

@magaldi_rodrigo

**Comment by SleeplessByte** [comment]
Nice. I'll tweet it out once it shows on my feed ;)

**Pull request merged**